### ISSUE TITLE
Reset does not reset terminal (zsh)

### ISSUE THREAD

**Issue by aral** [sedwards2009/extraterm#292]
![Screenshot from 2020-10-11 16-40-16](https://user-images.githubusercontent.com/41798/95682998-78ab3c80-0be0-11eb-9f67-1d369a69f711.png)

Reset merely shows that the result of the command is successful but it does not actually reset the terminal.

Also, running from source, I get the following error in the console:

```
Term.ts:2332 Unknown ESC control: .
error @ Term.ts:2331
_processDataEscape @ Term.ts:1601
_processWriteData @ Term.ts:902
_processOneWriteChunk @ Term.ts:749
_processWriteChunkRealTime @ Term.ts:713
(anonymous) @ Term.ts:671
setTimeout (async)
_scheduleProcessWriteChunk @ Term.ts:669
write @ Term.ts:651
(anonymous) @ Terminal.ts:284
(anonymous) @ EventEmitter.ts:37
fire @ EventEmitter.ts:37
_processPtyOutput @ PtyIpcBridge.ts:132
_handlePtyOutput @ PtyIpcBridge.ts:64
(anonymous) @ WebIpc.ts:66
handleAsyncIpc @ WebIpc.ts:65
emit @ events.js:223
onMessage @ electron/js2c/renderer_init.js:115
Term.ts:2333 console.trace
```

## System details

  - OS: Pop!_OS 20.04 LTS x86_64
  - GNOME Shell 3.36.4
  - Kernel: 5.4.0-7642-generic
  - Shell: zsh 5.8
  - DE: Unity
  - WM: Mutter
  - WM Theme: Adwaita
  - Theme: Pop-dark [GTK2/3
  - CPU: Intel i7-8550U (8) @ 4.000GHz
  - GPU: Intel UHD Graphics 620
  - Memory: 6305MiB / 15720MiB

**Comment by sedwards2009** [comment]
Great spot! It is also easily reproducible here too.

**Comment by aral** [comment]
@sedwards2009 For future reference, would you like me to close issues that have been fixed if I’ve reported them or would you like to do that yourself? (I’m happy to leave it to you, just wanted to make sure you weren’t expecting me to close it.) :)

**Comment by sedwards2009** [comment]
If you've resolved an issue yourself or no longer think it is needed, then do close it yourself and maybe say why.

For issues that have fixed in the code I tend to leave them open until the new release is actually available. The idea being that if you see that an issue is fixed then you know you have received it (assuming of course you are running the latest release).

That's the informal policy. It is not set in stone though.

**Comment by aral** [comment]
Sounds great and was what I’d assumed :)

**Comment by sedwards2009** [comment]
Released in version 0.54.0.
